Backpack/HorizontalNavigation/Classes/BPKHorizontalNavigationOption.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Backpack/Icon.h>
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface BPKHorizontalNavigationOption : NSObject
+
+@property(readonly, strong, nullable) BPKIconName iconName;
+@property(readonly, copy) NSString *name;
+@property(readonly) BOOL hasIcon;
+
+- (instancetype)initWithName:(NSString *)name;
+- (instancetype)initWithName:(NSString *)name iconName:(BPKIconName _Nullable)iconName;
```

Comment:
Is there a point in making the second argument `_Nullable` here? That's what the first initializer is for after all